packages/@aws-cdk/aws-eks/kubectl-layer/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
how much of a hit is this on our build time?

Reply:
2 minutes